egs/wsj/s5/steps/nnet3/make_denlats.sh

```diff
@@ -34,8 +34,8 @@ extra_left_context=0
 extra_right_context=0
 extra_left_context_initial=-1
 extra_right_context_final=-1
-feat_type=  # you can set this in order to run on top of delta features, although we don't
-            # normally want to do this.
```

Comment:
No need to have this variable any more.  This script doesn't need to support delta features or any features exept raw, and there is no need for a "feat_type" variable.